src/main/java/frc/trigon/robot/subsystems/elevator/ElevatorConstants.java

```diff
@@ -0,0 +1,167 @@
+package frc.trigon.robot.subsystems.elevator;
+
+import com.ctre.phoenix6.configs.TalonFXConfiguration;
+import com.ctre.phoenix6.controls.Follower;
+import com.ctre.phoenix6.signals.*;
+import edu.wpi.first.math.geometry.Pose3d;
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.math.geometry.Rotation3d;
+import edu.wpi.first.math.system.plant.DCMotor;
+import edu.wpi.first.units.Units;
+import edu.wpi.first.wpilibj.util.Color;
+import edu.wpi.first.wpilibj2.command.sysid.SysIdRoutine;
+import org.trigon.hardware.phoenix6.talonfx.TalonFXMotor;
+import org.trigon.hardware.phoenix6.talonfx.TalonFXSignal;
+import org.trigon.hardware.simulation.ElevatorSimulation;
+import org.trigon.utilities.mechanisms.ElevatorMechanism2d;
+
+public class ElevatorConstants {
+    private static final int
+            MASTER_MOTOR_ID = 13,
+            FOLLOWER_MOTOR_ID = 14;
+    private static final String
+            MASTER_MOTOR_NAME = "ElevatorMasterMotor",
+            FOLLOWER_MOTOR_NAME = "ElevatorFollowerMotor";
+    static final TalonFXMotor
+            MASTER_MOTOR = new TalonFXMotor(MASTER_MOTOR_ID, MASTER_MOTOR_NAME),
+            FOLLOWER_MOTOR = new TalonFXMotor(FOLLOWER_MOTOR_ID, FOLLOWER_MOTOR_NAME);
+
+    private static final NeutralModeValue NEUTRAL_MODE_VALUE = NeutralModeValue.Brake;
+    private static final InvertedValue
+            MASTER_MOTOR_INVERTED_VALUE = InvertedValue.Clockwise_Positive,
+            FOLLOWER_MOTOR_INVERTED_VALUE = InvertedValue.Clockwise_Positive;
+    private static final boolean FOLLOWER_MOTOR_OPPOSES_MASTER = false;
+    private static final double
+            P = 110,
+            I = 0,
+            D = 0,
+            KS = 0,
+            KV = 0,
+            KG = 0,
+            KA = 0;
+    private static final GravityTypeValue GRAVITY_TYPE_VALUE = GravityTypeValue.Elevator_Static;
+    private static final StaticFeedforwardSignValue STATIC_FEEDFORWARD_SIGN_VALUE = StaticFeedforwardSignValue.UseClosedLoopSign;
+    private static final Rotation2d FORWARD_HARD_LIMIT_ROTATIONS = Rotation2d.fromRotations(4);
+    private static final Rotation2d REVERSE_HARD_LIMIT_ROTATIONS = Rotation2d.fromRotations(0);
+    private static final double GEAR_RATIO = 5;
+    static final double
+            MOTION_MAGIC_CRUISE_VELOCITY = 25,
+            MOTION_MAGIC_ACCELERATION = 25,
+            MOTION_MAGIC_JERK = MOTION_MAGIC_ACCELERATION * 10;
+    static final boolean FOC_ENABLED = true;
+
+    private static final int MOTOR_AMOUNT = 2;
+    private static final DCMotor GEARBOX = DCMotor.getKrakenX60Foc(MOTOR_AMOUNT);
+    private static final double
+            MASS_KILOGRAMS = 9,
+            DRUM_RADIUS_METERS = 0.02,
+            RETRACTED_ELEVATOR_LENGTH_METERS = 0.2,
+            MAXIMUM_HEIGHT_METERS = 1.9,
+            MAXIMUM_LENGTH_METERS = MAXIMUM_HEIGHT_METERS - 0.05;
```

Comment:
the difference isn't very easy to understand here, the names aren't good 
and one isn't placed in the correct place inside of constants